sequencer/test/til/txs.go

```diff
@@ -233,7 +233,7 @@ func (tc *Context) generateBlocks() ([]common.BlockData, error) {
 			}
 		case common.TxTypeCreateVouch:
 			tx := common.L2Tx{
-				// Amount:      inst.Amount,
+				Amount: big.NewInt(0),
```

Comment:
For vouches will there be amount ?

Reply:
This is an temp thing initialising amount to 0, Done only because of conflicts will update once the structs on L2 txs are updated.